Mayhemfile

```diff
@@ -0,0 +1,9 @@
+project: fickling
```

Comment:
Can we change the target name to `fuzz_pickle_decompiler`?

Reply:
I fixed this